docs/source/interfaces/Engineering_Diffraction.rst

```diff
@@ -234,31 +246,38 @@ Output
 ^^^^^^
 
 Under the output section, the user is provided with an option of
-plotting data in three different formats. One Window - Replacing
-Plots: will replace the previous graph and plot a new graph on top.
-One Window - Waterfall: will plot all the generated focused
-workspace graphs in one window which can be useful while comparing
-various graphs. The Multiple Windows: will plot graph in
-separate windows. However, user may also change the Plot Data
-Representation drop-down box while a run is being carried out. This
-will update the interface and plot workspace according to the new
-given input. For example, if a user has selected One Window -
-Replacing Plots and then decides to change it to One Window -
-Waterfall during a run, the interface will carry on by plotting
+plotting data in three different formats. 
+
+- One Window - Replacing Plots: will replace the previous graph and plot a new graph on top.
+
+- One Window - Waterfall: will plot all the generated focused workspace graphs in one window 
+  which can be useful while comparing various graphs. 
+
+- Multiple Windows - will plot graph in separate windows. 
+
+However, user may also change the Plot Data representation drop-down box while a run is being carried out. This
```

Comment:
Removing [this line] (https://github.com/mantidproject/mantid/blob/b7be73a2d42de5e9f0a37741f7a9073d77087033/MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp#L633) should should enable it this again, but probably would require you to disable other features within `groupBox_focus_output_options` group box on the focus tab